zeppelin-interpreter/src/main/java/org/apache/zeppelin/scheduler/SchedulerFactory.java

```diff
@@ -63,11 +64,15 @@ public void destroy() {
   }
 
   public Scheduler createOrGetFIFOScheduler(String name) {
+    return createOrGetFIFOScheduler(name, this.executor);
```

Comment:
This would affect all the other interpreters. Since this is only for spark interpreter, we can create a specific scheduler for spark interpreter, and not change the existing `createOrGetFIFOScheduler`, `createOrGetParallelScheduler` and `createOrGetRemoteScheduler`

Reply:
I see. I will make another pull request by without touching existing function.